### PR TITLE
solved issue #58 (ReLU model)

### PR DESCRIPTION
**Pull request by KangHyeoni**
Resolves Issue #58 

### Description
ReLU model trained in 25 epochs and its accuracy on test data. Used Fashion mnist dataset.

### Technical Specifications
used keras, tensorflow, and matplotlib

### Checklist
- [x] Code compiles correctly.
- [x] Changes are tested properly.
- [x] Followed the Contributing guidelines.

### CODE REVIEW

**Comment by KangHyeoni** [comment]
@jkk2000 Could you review this PR.

**Comment by KangHyeoni** [comment]
I will request other PRs after this PR get accepted : )

**Review by jkk2000 — APPROVED** [review]
That's pretty good. You could go ahead and upload the rest.

**Comment by KangHyeoni** [comment]
@jkk2000 Finished all : )

**Comment by KangHyeoni** [comment]
@jkk2000 Could you merge this PR? Then I can work on another issue.
